src/shop.js

```diff
@@ -438,8 +439,11 @@ function BinartaShopjs(checkpoint, deps) {
             if (!fsm.getPaymentProvider())
                 if (shop.localStorage.getItem('binartaJSPaymentProvider'))
                     fsm.setPaymentProvider(shop.localStorage.getItem('binartaJSPaymentProvider'));
-                else
-                    fsm.setPaymentProvider('wire-transfer');
+                else {
+                    var profile = application.profile();
+                    if (profile.availablePaymentMethods && profile.availablePaymentMethods.length == 1)
+                        fsm.setPaymentProvider(profile.availablePaymentMethods[0]);
```

Comment:
When there's only one payment method available, there's no harm in selecting that one by default.
However, when there are more than one payment methods, we want to be sure the customer is aware of the selected payment method (so he has to select one himself). This is because there's no validation step after clicking "confirm".
When no payment method is selected and the customer clicks "confirm", the UI should display a helpful message. This is still todo.